python/hackathon_app/frontend/public/user.json

```diff
@@ -0,0 +1,4 @@
+{
+  "first_name": "Joel",
+  "last_name": "Dodge"
```

Comment:
Do you want this checked in?

Reply:
that's funny, I don't even know what this is! probably left over from some earlier iteration on something...

Reply:
ah, meredith added it, probalby for local testing the "thanks for registering" stuff